web_src/js/features/common-issue.js

```diff
@@ -43,6 +43,9 @@ export function initCommonIssue() {
       elementId = '';
       action = 'clear';
     }
+    if (action === 'toggle' && e.altKey) {
```

Comment:
I used alt since that was already used in the label filtering menu in the issue list. Shall I change that one as well?

Any preference for shift or ctrl?

Alternatively I can try to rid of the shortcut key and replace it with a little X button in the menu, maybe one that becomes visible on hover.

Reply:
I do no think  shift or ctrl is better, because  shift and ctrl are all for multiple selection.

`a little X button` seems the best one, but I am not sure whether it can be ready before 1.19 freezing (one day left)

Reply:
Since the altKey has been already used before, it's OK to continue to use it.

Leave other changes to a separate PR.

Reply:
I have some time today to look into this, though unsure if I will get it done in time. For me personally it doesn't matter if this ends up in 1.19 or 1.20.

I also wonder about what the best thing to do is regarding a11y. If there is something that would need to be done to ensure a user can get to that X button, or if it needs a completely different solution.

Reply:
Although my idea/approval doesn't count for review, I prefer to have PRs merged soon as it's ready, to avoid future conflicts.

And the "X Button" is not only for this PR, it's also for the legacy code,  so it really needs  a separate refactoring PR  IMO.

Reply:
Also, using this label menu with a keyboard is currently broken due to #22607.

Reply:
A quick patch for keyboard: #22607

Reply:
A quick patch for keyboard: Make issue label dropdown support Enter #23014